### PR TITLE
Docs: input data, pure fields and validation rules enricher

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: docs/guide/en/pure-fields.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Pure fields
```
[the remark this answers is not in the document]
Comment (arogachev) [reply docs/guide/en/pure-fields.md:1]: Not sure. I named it like in the [form](https://github.com/yiisoft/form/blob/master/docs/guide/en/pure-fields.md) package. Just "field" can be confusing with built-in fields. Maybe rename it to something more generic like "Displaying fields"?

File: docs/guide/en/pure-fields.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+Field::text($formModel, 'login', theme: 'my-theme');
+
+// Using factory
+
+$factory = new FieldFactory();
+$factory->text('name', 'value', theme: 'my-theme');
+```
+
+To apply the theme for all fields, either pass it as argument in constructor (supported by factory).
```
[the remark this answers is not in the document]
Comment (samdark) [reply docs/guide/en/pure-fields.md:64]: It doesn't work like that as far as I remember. 2nd part could start with "As an alternative," or something.

File: docs/guide/en/pure-fields.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samdark) [reply docs/guide/en/pure-fields.md:1]: Yes. Should be better with "Displaying fields".

File: docs/guide/en/pure-fields.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (arogachev) [reply docs/guide/en/pure-fields.md]: Nope. But I rewrote it without using "either".

**Pull request merged**